<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultWindow</class>
 <widget class="QWidget" name="ResultWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labIndicator_num">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>60</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labIndicator">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>60</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/indicator_off);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDecCount">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>85</width>
     <height>85</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnIncCount">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>270</y>
     <width>85</width>
     <height>85</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelShow">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>250</width>
     <height>46</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/label_long);</string>
   </property>
   <property name="text">
    <string>Сообщение</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>517</x>
     <y>156</y>
     <width>943</width>
     <height>851</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="widgetResizable">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignTop</set>
   </property>
   <widget class="QWidget" name="resultAreaWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>943</width>
      <height>0</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>411</width>
       <height>401</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="resultVbox">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="labelFrame">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>110</y>
     <width>1090</width>
     <height>930</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/frameBig);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/backgroundSteel);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btnUpdate">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>750</y>
      <width>95</width>
      <height>95</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelCount">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>670</y>
      <width>250</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/label_long);</string>
    </property>
    <property name="text">
     <string>Показать</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelTitle">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>1071</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color: rgb(55, 55, 55);</string>
    </property>
    <property name="text">
     <string>Просмотр результатов</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>1590</x>
      <y>940</y>
      <width>95</width>
      <height>95</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelExit">
    <property name="geometry">
     <rect>
      <x>1710</x>
      <y>960</y>
      <width>200</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/label1);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <zorder>frame</zorder>
  <zorder>btnDecCount</zorder>
  <zorder>btnIncCount</zorder>
  <zorder>labIndicator</zorder>
  <zorder>labIndicator_num</zorder>
  <zorder>labelFrame</zorder>
  <zorder>labelShow</zorder>
  <zorder>scrollArea</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
